src/services/message.service.ts

```diff
@@ -31,6 +31,55 @@ export class MessageService {
     this._sendConstructedReport(report, { files: message.attachments.map((e) => e.url) });
   }
 
+  async sendTextMessage(
+    message: IMessage,
+    content: string,
+    argOptions: ITextMessageOptions
+  ): Promise<void> {
+    const options: ITextMessageOptions = {};
+    options.header = argOptions.header ? `${argOptions.header}\n` : '';
+    options.footer = argOptions.footer ? `\n${argOptions.footer}` : '';
+    options.reply = argOptions.reply || false;
+    options.delimiter = argOptions.delimiter === '*' ? '' : argOptions.delimiter;
```

Comment:
wont this sometimes be undefined?

Reply:
If you're u mean delimiter, yes it can at which point it's either not used, or is caught and a error is thrown on line 51.  If it's any other the other ones I thought the above code would deal with it.